src/layouts/Layout.astro

```diff
@@ -24,24 +31,342 @@ import "../styles/globals.css";
     <meta property="twitter:card" content="summary_large_image" />
 
     <link
-      rel="shortcut icon"
-      href="https://cdn.purplebubble.org/logo.png"
-      type="icon/png"
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="/favicon/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
       sizes="32x32"
+      href="/favicon/favicon-32x32.png"
     />
     <link
-      rel="stylesheet"
-      href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
-      data-tag="font"
+      rel="icon"
+      type="image/png"
+      sizes="194x194"
+      href="/favicon/favicon-194x194.png"
     />
     <link
-      rel="stylesheet"
-      href="https://unpkg.com/@teleporthq/teleport-custom-scripts/dist/style.css"
+      rel="icon"
+      type="image/png"
+      sizes="192x192"
+      href="/favicon/android-chrome-192x192.png"
     />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="/favicon/favicon-16x16.png"
+    />
+    <link rel="manifest" href="/favicon/site.webmanifest" />
+    <link rel="shortcut icon" href="/favicon/favicon.ico" />
+    <meta name="apple-mobile-web-app-title" content="Purple Bubble" />
+    <meta name="application-name" content="Purple Bubble" />
+    <meta name="msapplication-TileColor" content="#9f00a7" />
+    <meta
+      name="msapplication-TileImage"
+      content="/favicon/mstile-144x144.png"
+    />
+    <meta name="msapplication-config" content="/favicon/browserconfig.xml" />
+    <meta name="theme-color" content="#ffffff" />
```

Comment:
do we want this theme color to be purple?

Reply:
does #8100e5 work?

Reply:
yep!